TypeScriptFrontend/src/game/objects/otherPlayer.ts

```diff
@@ -20,5 +27,58 @@ export class OtherPlayer {
     set color(color: readonly [number, number, number]) {
         this._color = color;
     }
+
+    destroyRenderedText() {
+        this.renderedText.forEach((text) => text.destroy())
+    }
+
+    //proposal: remove 'scene' parameter and require 'scene' in constructor
```

Comment:
Yea this makes. I guess its used in a few different functions. I think theres some other classes that would benefit from this change as well.